### PR TITLE
sub account creation helper function

### PR DESCRIPTION
**Pull request by jheld**
looking to add support for creation of sub accounts (maybe other operations, too, but starting with creation).

This is for issue #146

### CODE REVIEW

**Comment by steverecio** [comment]
LGTM. Might make sense to move `create_sub_account` and `discover_twilio_credentials` to static functions on the Credential model at some point but this is consistent at least.

Not sure if unit tests make sense here or not given the reliance on the twilio lib.

**Comment by jheld** [comment]
@steverecio right, agreed it could be nice having this tied more directly against `Credential` (or those could be wrappers). Yeah, let's see how this works!

OK, I suppose the first thing I should do is actually try this out! I'll try to communicate my results back here within the next couple weeks.

**Comment by DmytroLitvinov** [comment]
When we could merge it guys and publish to PyPi?

**Comment by DmytroLitvinov** [comment]
Friendly ping.

If we need to refactor something - I am ready to do that.

**Comment by jheld** [comment]
version `0.13.1.a0` is now on PyPI.

File: django_twilio/utils.py [hunk 1]
```diff
@@ -91,5 +91,22 @@ def get_blacklisted_response(request):
...
 
 
+def create_sub_account(user, twilio_client, friendly_name=None):
+    new_account = twilio_client.api.accounts.create(friendly_name=friendly_name or user)
+    new_credential = Credential.objects.create(user=user, account_sid=new_account.sid, auth_token=new_account.auth_token, name=new_account.friendly_name)
+    return new_credential, new_account
+
+
+def close_sub_account(account_sid, twilio_client):
+    return twilio_client.accounts(account_sid).update(status="closed")
```
Comment (DmytroLitvinov) [inline django_twilio/utils.py:101]: Hi @jheld , 

I guess it should be also`twilio_client.api.accounts`?

File: django_twilio/utils.py [hunk 2]
```diff
@@ -91,5 +91,22 @@ def get_blacklisted_response(request):
...
+
+
+def close_sub_accounts_for_user(user, twilio_client):
+    accounts = []
+    for credential_obj in Credential.objects.filter(user=user):
+        accounts.append(twilio_client.accounts(credential_obj.account_sid).update(status="closed"))
```
Comment (DmytroLitvinov) [inline django_twilio/utils.py:107]: Same probably?

File: django_twilio/utils.py (hunk 1 above)
Comment (jheld) [reply django_twilio/utils.py:101]: I did test this locally so I don't think it was breaking but if you'd like to make a PR to change it (or fix it), please do! I've just released `0.13.1b0` so there's a small window of time before `0.13.1` officially.

**Pull request merged**